### DISCUSSION TITLE
Please post your discussions on the Hazelcast Community Slack: https://slack.hazelcast.com/

### DISCUSSION THREAD

**Discussion by JackPGreen** [hazelcast/hazelcast#26312] [General]
This discussions portal is read only, please post continue discussions on the Hazelcast Community Slack: https://slack.hazelcast.com/

**Comment by marcoschulte** [comment]
Hi @JackPGreen 
unfortunately the Slack invite seems to be expired.
Regards
Marco

**Comment by fawazghali** [comment]
Please use the following link:

https://join.slack.com/t/hazelcastcommunity/shared_invite/zt-2xi772zes-iEKt~k4OduV8PhWOc81~Xg

We're working on updating the main link: https://slack.hazelcast.com/
